spec/features/user_story_spec.rb

```diff
@@ -0,0 +1,67 @@
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+# As a customer
+# So that I can order the meal I want
+# I would like to be able to select some number of several available dishes
+
+# As a customer
+# So that I can verify that my order is correct
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+# As a customer
+# So that I am reassured that my order will be delivered on time
+# I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+require 'dish'
+require 'menu'
+require 'order'
+
+describe 'user stories' do
+
+  it 'so that orders can be placed display a list of dishes' do
+    dish1 = Dish.new("Chicken", 2.99)
```

Comment:
Consider using `let` to set up your tests with less repetition.  http://www.betterspecs.org/#let is pretty good for examples. 👍